g2p/apply.py

```diff
@@ -83,12 +129,22 @@ def run(self):
                         "-m",
                         self.g2p_model.get_path(),
                         "-a",
-                        self.word_list.get_path(),
+                        word_list_path,
                     ],
                     stdout=out,
                     stderr=err,
                 )
 
+    def merge(self):
+        g2p_lex_file_list = " ".join(map(tk.uncached_path, self.out_g2p_lexicon_parts))
+        self.sh(f"cat {g2p_lex_file_list} > {tk.uncached_path(self.out_g2p_lexicon)}")
+        g2p_untranslated_file_list = " ".join(
+            map(tk.uncached_path, self.out_g2p_untranslated_parts)
+        )
+        self.sh(
+            f"cat {g2p_untranslated_file_list} > {tk.uncached_path(self.out_g2p_untranslated)}"
+        )
```

Comment:
we want to get rid of `tk.uncached_path`. In your case it is always save to use `get_path` instead.

Reply:
Removed all instances of tk.uncached_path now.